.github/workflows/CI-build-test.yml

```diff
@@ -1,69 +1,65 @@
-name: CI-build-test
+name: Build-and-Test
 
-run-name: "Workflow performing CI steps: build, unit testing, integration testing"
+run-name: "Workflow CI/CD Steps: build, unit and integration testing"
 
 on:
   workflow_call:
 
 env:
-  RUST_BACKTRACE: 1
+  IMAGE_NAME: zencash/sc-ci-base
```

Comment:
Very minor question, is it fine we are using this image named `zencash`? Do we want to duplicate it with different name?

Reply:
This is a good question. We build these images under zencash workspace. I will need @lander86 and @cronicc input on it.

Reply:
@JackPiri those are public images with the open source content. Should be OK to use them.

Reply:
As just mentioned by @ysibirski , it's ok to use those image.